src/app/app-routing.module.ts

```diff
@@ -2,7 +2,8 @@ import {NgModule} from '@angular/core';
 import {Routes, RouterModule} from '@angular/router';
 
 const routes: Routes = [
-  {path: '', loadChildren: './tabs/tabs.module#TabsPageModule'}
+  {path: '', loadChildren: './tabs/tabs.module#TabsPageModule'},
+  { path: 'restaurant', loadChildren: './restaurant/restaurant.module#RestaurantPageModule' }
```

Comment:
```suggestion
  { path: 'restaurant', loadChildren: './restaurant/restaurant.module#RestaurantPageModule' },
```
zostawianie tych przecinków jest uważane za dobrą praktykę bo przy następnym MR mniej linii się zmieni